cves/CVE-2010-1157.yml

```diff
@@ -44,36 +44,33 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  A portion of Tomcat that handles http authentication via DIGEST and BASIC strategies had the potential to allow
-  for the viewing of the local name or IP address of the server running Tomcat. The header for the authentication 
-  via these strategies includes a <realm-name> element.
-  If a <realm-name> element is specified for the application in one of the configuration files, it will be used. 
-  However, if a <realm-name> is not specified then Tomcat will generate a realm name and
-  in some circumstances this can expose the local host name or IP address of the machine running Tomcat. 
+  May allow remote attackers to discover the server's hostname or IP address by 
+  sending a request for a resource that requires one of two specific 
+  authentication levels.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2010-04-23
+  announced:
   url:
 bugs: []
 fixes_vcc_instructions: |
   Please put the Git commit SHA in "commit" below, and any notes about how this
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: 78e45a225c5c5564945191f7e421f8859885d63b
-     note: SVN rev 936541, from the Tomcat website.
-   - commit:
-     note:
+   - commit: a4de93b2aaf74fff7c1e194f7ce3d2228a9680b8
+     note: |
+      SVN rev 936539, from the Tomcat website. The fix was to add a variable for
+      the default behavior of the realm.
```

Comment:
Using another word besides "realm" would help the reader understand what that means exactly. I worked on the same CVE so I understand the context, but others who haven't may not.

Reply:
I have updated this so that it understandable to a larger audience.